client/wfm/src/result/WorkOrderResult.ts

```diff
@@ -0,0 +1,11 @@
+import {StepResult} from './StepResult';
+export interface WorkOrderResult {
```

Comment:
In the original format this was `Workorder` but I'm happy to have it this way for TypeScript stuff.